c2rust-analyze/src/rewrite/expr/convert.rs

```diff
@@ -368,6 +186,11 @@ impl<'a, 'tcx> intravisit::Visitor<'tcx> for HirRewriteVisitor<'a, 'tcx> {
             }
         };
 
+        for mir_rw in mir_rws {
+            hir_rw = rewrite_from_mir_rws(&mir_rw, hir_rw);
+        }
+
+        /*
```

Comment:
Is this going to be used later?  Is there a reason it's just commented out vs. removed?

Reply:
Removed.  I had that commented out initially because I thought I might need to port that logic over to the new design, and forgot to delete it afterward.